WordPress/Classes/ViewRelated/Post/PostSharingController.swift

```diff
@@ -104,12 +104,15 @@ import SVProgressHUD
     }
 
     func shareReaderPost(post: ReaderPost, fromView anchorView:UIView, inViewController viewController:UIViewController) {
-
+        var permalink = post.permaLink ?? ""
```

Comment:
Should an empty string be set here in case of nil? Or continue to be nil instead? This is later passed to the optional `link` for `sharePost`.